src/main.ts

```diff
@@ -9,9 +9,15 @@ const cpExec = util.promisify(require('child_process').exec);
 import { createScriptFile, TEMP_DIRECTORY, NullOutstreamStringWritable, deleteFile, getCurrentTime, checkIfEnvironmentVariableIsOmitted } from './utils';
 
 const START_SCRIPT_EXECUTION_MARKER: string = "Starting script execution via docker image mcr.microsoft.com/azure-cli:";
-const AZ_CLI_VERSION_DEFAULT_VALUE = 'agentazcliversion'
+const AZ_CLI_VERSION_DEFAULT_VALUE = 'agentazcliversion';
+const prefix = !!process.env.AZURE_HTTP_USER_AGENT ? `${process.env.AZURE_HTTP_USER_AGENT}` : "";
 
 export async function main() {
+    let usrAgentRepo = `${process.env.GITHUB_REPOSITORY}`;

```

Comment:
Is `GITHUB_REPOSITORY` sensitive? Can we log this in the telemetry?

Reply:
@Jacekey23 @dcaro Could you help us to confirm this?

Reply:
@Jacekey23 @dcaro I have encoded the user repo with hash. Is it appropriate to store the information of user repo with hash?